pyiron_contrib/protocol/jobs/decoupled_oscillators.py

```diff
@@ -253,18 +282,12 @@ def from_hdf(self, hdf=None, group_name=None):
             group_name (str): HDF5 subgroup name - optional
         """
         super(DecoupledOscillators, self).from_hdf()
-        try:
-            self.input.ref_job
-        except AttributeError:
-            ref_job = self.pop()
-            self.input.from_hdf(self.project_hdf5)
-            self.input.ref_job = ref_job
+        self.input.from_hdf(self.project_hdf5)
+        self.input.ref_job = self[self.input.ref_job_name]
         self.output.from_hdf(self.project_hdf5)
 
     def interactive_close(self):
-        self[0].interactive_close()  # close the base job
-        self.to_hdf()   # run to_hdf to re-save input
-        self.output.to_hdf(self.project_hdf5)  # save output
+        self[self._base_name].interactive_close()  # close the base job
```

Comment:
This bit still complains that 'self[self._base_name]' is a 'ProjectHDFio' object.